libs/build_jasper.sh

```diff
@@ -56,13 +56,18 @@ else
     useCmake=NO
 fi
 
+if [[ "${STACK_jpeg_build}" == "YES" ]]; then
+    module load jpeg
```

Comment:
Would just a `module try-load jpeg` work? Because if one installed jpeg previously and now just updating jasper, the if block would return false.

Reply:
Yes, I came across that only later when I looked at the NCEPLIBS scripts. Should be fixed now, please check.